openvdb_houdini/houdini/SOP_OpenVDB_Scatter.cc

```diff
@@ -164,6 +180,12 @@ newSopOperator(OP_OperatorTable* table)
             "If enabled, scatter points in the interior region of a level set."
             " Otherwise, scatter points only in the narrow band."));
 
+    // an iso-surface offset for interior scattering
+    parms.add(hutil::ParmFactory(PRM_FLT_J, "offset", "Offset")
+        .setDefault(PRMzeroDefaults)
+        .setRange(PRM_RANGE_UI, -1.0f, PRM_RANGE_UI, 1.0f)
+        .setTooltip("The offset at which to interpret the iso surface."));
```

Comment:
I don't feel this is a common enough operation to warrant a new option on the scatter - you can just use a VDB Reshape SDF / OpenVDB Offset Level Set beforehand with minor slowdown.

Reply:
I disagree - This allows you to scatter within an offset interior mask without changing the SDF. Without this option you'd need to deep copy the SDF and manipulate its values. There are no public tools to extract the interior mask currently within Houdini. We do a lot of scattering which we expect to by up to the iso-surface of the SDF, this feels like quite an intuitive option seeing as the code has to call sdfInteriorMask anyway.